ci/L0_additional_outputs_vllm/additional_outputs_test.py

```diff
@@ -76,6 +84,16 @@ def _get_inputs(
                 np.array([return_cumulative_logprob], dtype=bool)
             )
 
+        if return_logprobs is not None:
+            inputs.append(grpcclient.InferInput("return_logprobs", [1], "BOOL"))
```

Comment:
Let's be consistent with TRTLLM and use `return_log_probs`: https://github.com/triton-inference-server/tensorrtllm_backend/blob/91c07d3acd865238176f0fcc3375155e3085c5c8/all_models/inflight_batcher_llm/ensemble/config.pbtxt#L442